cumulusci/tasks/apex/testrunner.py

```diff
@@ -210,6 +213,18 @@ def _init_options(self, kwargs):
 
         self.counts = {}
 
+        if "required_org_code_coverage_percent" in self.options:
+            try:
+                self.code_coverage_level = int(
+                    self.options["required_org_code_coverage_percent"]
+                )
+            except ValueError:
+                raise TaskOptionsError(
+                    f"Invalid code coverage level {self.options['required_org_code_coverage_percent']}"
+                )
+        else:
+            self.code_coverage_level = None
```

Comment:
I really need to find time to review Paul's PR adding declarative option validation using pydantic...